if gss is disabled, then svc_auth_gss_ops won't be available

--- src/svc.c
+++ src/svc.c
@@ -77,7 +77,7 @@
 
 extern rwlock_t svc_lock;
 extern rwlock_t svc_fd_lock;
-extern struct svc_auth_ops svc_auth_gss_ops;
+extern struct svc_auth_ops svc_auth_gss_ops __attribute__((weak));
 
 static struct svc_callout *svc_find (rpcprog_t, rpcvers_t,
 				     struct svc_callout **, char *);
